queries.sql

```diff
@@ -8,3 +8,42 @@ SELECT name, escaped_attempts FROM animals WHERE weight_kg > 10.5
 SELECT * FROM animals WHERE neutered = true
 SELECT * FROM animals WHERE name != 'Gabumon' OR SELECT * FROM animals WHERE name <> 'Gabumon'
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3
+
+/* DAY TWO*/
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals
+
+ROLLBACK;
+SELECT * FROM animals
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '01-01-2022';
```

Comment:
- [ ] Great job 👍🏻  you only need to update the date data type to follow the format YYYY-MM-DD.